router/router.go

```diff
@@ -146,7 +146,8 @@ type routerRequest struct {
 }
 
 // semi-unmarshalls message, returns method name
-func (r *Router) getRequest(payload []byte, context transports.MessageContext) (request routerRequest, err error) {
+func (r *Router) getRequest(
```

Comment:
Let's unify syntax model. We don't usually do this, instead:
```
getRequest( some parameters....,
      some more parameters....) {
```
So split the line in two or three lines of similar size.